source/geometries/Next100Vessel.cc

```diff
@@ -342,16 +342,24 @@ namespace nexus {
     } else if  (gas_ == "XeHe") {
       vessel_gas_mat = materials::GXeHe(pressure_, 300. * kelvin,
 					    xe_perc_, helium_mass_num_);
+    } else if  (gas_ == "GAr") {
+      vessel_gas_mat = materials::GAr(pressure_, temperature_);
     } else {
       G4Exception("[Next100Vessel]", "Construct()", FatalException,
 		  "Unknown kind of xenon, valid options are: "
-                  "natural, enriched, depleted, or XeHe.");
+                  "naturalXe, enrichedXe, depletedXe, GAr, or XeHe.");
     }
 
+    // Gas Properties
+    if (gas_ == "GAr"){
+      vessel_gas_mat->SetMaterialPropertiesTable(opticalprops::GAr(sc_yield_, e_lifetime_));
+    }
+    else {
     vessel_gas_mat->SetMaterialPropertiesTable(opticalprops::GXe(pressure_,
                                                                  temperature_,
                                                                  sc_yield_,
                                                                  e_lifetime_));
+    }
```

Comment:
This is an issue for a different PR, but it seems that we are defining the same gas in different places, perhaps we should revisit this and propagate the gas material from the top down.